functions/src/tebra-sync/__tests__/tebraProxyClient.test.js

```
@@ -0,0 +1,443 @@
+    GoogleAuth.mockImplementation(() => mockAuth);
+    
+    // Reset client state
+    tebraProxyClient.initialized = false;
+    
+    // Set up environment
+    process.env.TEBRA_CLOUD_RUN_URL = 'https://test-service.run.app';
+    process.env.TEBRA_INTERNAL_API_KEY = 'test-api-key';
+  });
+
+  afterEach(() => {
+    delete process.env.TEBRA_CLOUD_RUN_URL;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)